### PR TITLE
Add `as_chrono_duration()` utility

### LINKED ISSUE #187: Add utility to convert Au time quantity to chrono duration
We could call this `to_chrono()` or `as_chrono()`.

### PR DESCRIPTION
**Pull request by chiphogg**
This makes it easy to convert any Au duration quantity to an
exactly-equivalent `std::chrono::duration`.

Fixes #187.

### CODE REVIEW

File: au/au.hh [hunk 1]
```diff
@@ -34,4 +34,14 @@ struct CorrespondingQuantity<std::chrono::duration<RepT, Period>> {
     static constexpr ChronoDuration construct_from_value(Rep x) { return ChronoDuration{x}; }
 };
 
+// Convert any Au duration quantity to an equivalent `std::chrono::duration`.
```
Comment (geoffviola) [inline au/au.hh:37]: Q: au.hh's two functions seem to be `std::chrono` related. Is it worth breaking that out into its own files?

**Review by geoffviola — APPROVED** [review]
Implementation and test look good.

File: au/au.hh (hunk 1 above)
Comment (chiphogg) [reply au/au.hh:37]: That's a good point.  I'll do that in a follow-up PR.

Comment (chiphogg) [reply au/au.hh:37]: #201.

**Pull request merged**